hypertrace-graphql-explorer-schema/build.gradle.kts

```diff
@@ -7,6 +7,7 @@ dependencies {
   api("com.graphql-java:graphql-java")
   api("org.hypertrace.core.graphql:hypertrace-core-graphql-spi")
   api("io.github.graphql-java:graphql-java-annotations")
+  api(project(":hypertrace-graphql-metric-schema"))
```

Comment:
metric schema is part of the explorer api, was incorrectly categorized before (albeit not strictly related)